.changeset/itchy-bears-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
```suggestion
"@primer/react": minor
```

With what we're adding into SelectPanel what do you think about having this as a `minor` since we're adding in functionality?